src/lib/core/a11y/focus-trap.html

```diff
@@ -1,3 +1,3 @@
 <div *ngIf="!disabled" tabindex="0" (focus)="focusLastTabbableElement()"></div>
-<div #trappedContent><ng-content></ng-content></div>
+<div #trappedContent class="focus-trap-content"><ng-content></ng-content></div>
```

Comment:
Also prefix on `.focus-trap-content`. This is as good a time as any to introduce the `cdk-` prefix.